### PR TITLE
Added homebrew cask executable path, switched to dirs crate for home_dir

### PR DESCRIPTION
**Pull request by obojdi**
Hey. Homebrew installs Sonic-PI cask with server executable to its own location, so here's the path. To make it work, you also have to change permissions for `sonic-pi-server.rb` with `chmod +x`

Also to get rid of compiler warnings, env::home_dir was replaced with dirs::home_dir from `dirs` crate, and `error.description` with `error.to_string`. Thus, now unused `std::error::Error` was commented out.

### CODE REVIEW

File: src/file.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use std::error::Error;
+// use std::error::Error;
```
Comment (lpil) [inline src/file.rs]: Would you mind deleting this line?

**Comment by lpil** [comment]
Great! Thank you! I've left one small note above

File: src/file.rs (hunk 1 above)
Comment (obojdi) [reply src/file.rs]: Sure thing

Comment (lpil) [reply src/file.rs]: Seems you've uncommented it rather than delete it?

Comment (obojdi) [reply src/file.rs]: Yep seems I've got you wrong =) gone now

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**